QuickCheck.cabal

```diff
@@ -61,15 +61,8 @@ flag templateHaskell
   Description: Build Test.QuickCheck.All, which uses Template Haskell.
   Default: True
 
-flag light
-  Description:
-    Build with as few dependencies as possible.
-    Some instances will be missing, but build time will be reduced.
-    Can be useful on e.g. Travis.
-  Default: False
-
 library
-  Build-depends: base >=4.3 && <5, random, containers, array
+  Build-depends: base >=4.3 && <5, random, containers
```

Comment:
Just a note: `containers` depends on `array` so we don't save anything by removing the array instances. On the other hand, since those instances were already in `quickcheck-instances` perhaps it's the right thing to do anyway.